extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -70,7 +71,7 @@ export class GoogleSubscriptionsPlatform {
     this.serviceAdapter_ = serviceAdapter;
     /** @private @const {!ConfiguredRuntime} */
     this.runtime_ = new ConfiguredRuntime(
-        ampdoc.win,
+        new DocImpl(ampdoc),
```

Comment:
I guess a small test that we're not using ampdoc directly anymore should be needed here

Reply:
Done. I wish we could stub constructors :(